README.md

```diff
@@ -205,6 +205,7 @@ Loads a document passed using `file` prop.
 |onPassword|Function called when a password-protected PDF is loaded. Defaults to a function that prompts the user for password.|`(callback) => callback('s3cr3t_p4ssw0rd')`|
 |onSourceError|Function called in case of an error while retrieving document source from `file` prop.|`(error) => alert('Error while retrieving document source! ' + error.message)`|
 |onSourceSuccess|Function called when document source is successfully retrieved from `file` prop.|`() => alert('Document source retrieved!')`|
+|onTaskCreated|Function called when a task to load a document has been created.|`(task) => if (queue.isFull) { return task.cancel(); } else { return queue.promise; }`|
```

Comment:
Returning anything here doesn't make any sense and its purpose may be confusing for the users. Perhaps something like this?

```suggestion
|onTaskCreated|Function called when a task to load a document has been created.|`(task) => if (queue.isFull) { task.cancel(); }`|
```